.github/workflows/deploy.yml

```
@@ -46,11 +45,22 @@ jobs:
+            sudo yum update -y && sudo yum install -y java-17-amazon-corretto
+            # 기존 java -jar 프로세스 모두 강제 종료
             for pid in $(pgrep java); do
               if ps -p $pid -o args= | grep -q 'java -jar'; then
-                echo "Java process with 'java -jar' found (PID: $pid). Terminating..."
+                echo "Terminating Java process (PID: $pid)"
                 kill -9 $pid
               fi
             done
-            echo "nohup java -jar ~/*.jar > ~/app.log 2>&1 &" | at now
\ No newline at end of file
+            
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)